src/NServiceBus.Core/Outbox/OutboxSettings.cs

```diff
@@ -1,28 +1,27 @@
 ﻿namespace NServiceBus.Outbox
 {
     using System;
+    using NServiceBus.Configuration.AdvanceExtensibility;
     using Settings;
 
     /// <summary>
     /// Custom settings related to the outbox feature
     /// </summary>
-    public class OutboxSettings
+    public class OutboxSettings : ExposeSettings
```

Comment:
I've provided an example of what the goal is in https://github.com/Particular/NServiceBus/pull/2712#discussion_r28473445